global/classic-zaza/pip.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
If you grep this repo recursively for `tox.ini` you'll see that we have some scripts for cloning all our charms and copying this boilerplate to them all. ([Example](https://github.com/openstack-charmers/release-tools/blob/master/batch-example#L18)) I believe `pip.sh` should be added to all these mechanisms.

If @ajkavanagh agrees I believe it doesn't need to be a new flag for these scripts. Instead it can just fall under the same `--update-tox` flag.

Reply:
added `pip.sh` to be copied when running `_update-tox-files`.